app/routes/__index/index.tsx

```diff
@@ -21,32 +21,38 @@ export default function IndexRoute() {
         paddingY={{ base: '2', md: '7' }}
         gap={{ md: '16' }}
       >
-        <Flex width={{ base: '100%', md: '50%' }} textAlign="center" padding="5">
-          <Text fontSize={{ base: 'sm', md: 'lg' }}>
-            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum
-            has been the industry's standard dummy text ever since the 1500s, when an unknown
-            printer took a galley of type and scrambled it to make a type specimen book.
+        <Flex
+          width={{ base: '100%', md: '75%' }}
+          textAlign="center"
+          padding="5"
+          flexDirection="column"
+        >
+          <Heading size="lg">Welcome to My.Custom.Domain!</Heading>
+          <Text fontSize={{ base: 'sm', md: 'lg' }} mt={4}>
+            My Custom Domain creates a unique web address for your project that makes it easier to
+            access on the internet. My Custom Domain also generates a HTTPS certificate to provide a
+            secure connection between client and your domain.
           </Text>
         </Flex>
       </Flex>
       <Flex
         flexDirection={{ base: 'column', sm: 'row' }}
         justifyContent="center"
-        paddingY="10"
+        paddingY="5"
         paddingX={{ base: '10', md: '5' }}
         gap={{ base: '5', lg: '10' }}
       >
         <LandingPageCard
           path="/dns-records"
           pathName="Manage DNS Records"
           cardName="DNS Records"
-          cardDescription="DNS Record: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. "
+          cardDescription="DNS Record is a type of data stored in Domain Name System server that maps a domain name to an IP address, or another domain, or other information.  You can create a unique custom domain for each of your projects. My Custom Domain makes this process so simple and easy. You just need to submit a simple form to create your own domain. "
```

Comment:
"...in the Domain Name System that maps a domain name to an IP address, another domain, or records text information....My.Custom.Domain makes this process simple and easy."

I would drop the last sentence.

Reply:
Yeah, I tried to balance the amount of text on both boxes. I guess it doesn't look too bad. 

<img width="1512" alt="Screenshot 2023-04-04 at 6 50 37 PM" src="https://user-images.githubusercontent.com/59734889/229939361-1f447b1f-05dd-4dd9-99c2-a6bb5a94bddd.png">

Reply:
I think you can set a minHeight for the Flex component so regardless of the amount of characters they will be equal in length? Also, if you want to make them perfectly align you can use font-style `mono`, but I'll let you decide if it looks good